frontend/src/services.js/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+
+const baseUrl = "http://localhost:8000/api";
+
+export const getCourses = async() => {
+    try{
+        const response = await axios.get(`${baseUrl}/course/public`);
+        return response.data
```

Comment:
I don't remember the interaction here, but does correctly jump to the catch block if we get a 400/500 error code?